easybuild/easyblocks/p/psmpi.py

```diff
@@ -57,6 +57,7 @@ def extra_options(extra_vars=None):
         extra_vars.update({
             'mpich_opts': [None, "Optional options to configure MPICH", CUSTOM],
             'threaded': [False, "Enable multithreaded build (which is slower)", CUSTOM],
+            'pscom_allin_path': [None, "Enable pscom integration by giving its source path", CUSTOM],
```

Comment:
`s/source path/installation prefix/` ?

Reply:
Well, it is an installation prefix just if it is installed, right? But with the `allin` option it is not really installed so....... I still think source path is "correcter"